### PR TITLE
[CSL-1009] - Move gitHash and gitBranch printing to executables

### PR DESCRIPTION
**Pull request by 7sedam7**
It was building, tests were passing
after merge of master, I can not build any more, so if it doesn't build please let me know

### CODE REVIEW

File: src/addr-convert/Main.hs [hunk 1]
```diff
@@ -81,3 +84,7 @@ fromAvvmPk addrText = do
 
 convertAddr :: Text -> IO Text
 convertAddr addr = pretty . makeRedeemAddress <$> fromAvvmPk (toText addr)
+
+
+main :: IO ()
```
Comment (gromakovsky) [inline src/addr-convert/Main.hs]: Why did you add one more `main` here?

File: src/node/Main.hs [hunk 2]
```diff
@@ -4,18 +4,20 @@
 {-# LANGUAGE RankNTypes          #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeOperators       #-}
+{-# LANGUAGE TemplateHaskell     #-}
 
 module Main
        ( main
        ) where
 
+import           Development.GitRev  (gitBranch, gitHash)
```
Comment (gromakovsky) [inline src/node/Main.hs]: According to the [style-guide](https://github.com/serokell/serokell-util/blob/master/serokell-style.md) `Universum` should be a separate block of imports and 3rd-party libraries should be all in the same block.

**Comment by gromakovsky** [comment]
@7sedam7 FYI: I force-pushed to this branch, reset your local version please.

**Comment by gromakovsky** [comment]
I think it can be closed, because we need a different solution anyway, see https://issues.serokell.io/issue/CSL-1328#comment=96-5292

**Comment by ChShersh** [comment]
Agree. If we need different solution so why spend time on non-viable solution? Does it improve anyone's life?

**Comment by gromakovsky** [comment]
Well, this solution still improves our lifes, because it makes compilation faster than what we currently have. So if we know that we will not do CSL-1328 in near future, we can finish this effort at least (which seems much easier than CSL-1328).
